src/main/java/org/opentripplanner/routing/api/request/RouteRequest.java

```diff
@@ -53,6 +56,8 @@ public class RouteRequest implements Cloneable, Serializable {
 
   private GenericLocation to;
 
+  private List<StopLocations> passthroughLocations = Collections.emptyList();
```

Comment:
Consider:
```suggestion
  private List<PassThroughPoint> passThroughPoints = List.of();
```

Then the PassThroughPoint class should be in the same package as RouteRequest:

```
class PassThroughPoint {
    /** Optional for logging/debugging only */
    name : String
    /** at least one point must exist */
    locations : List<StopLocation>
  }
  ```